### PR TITLE
Feature add bottom navbar

### LINKED ISSUE #65: Make bottom button-navigation
- [ ] Add bottom navigation
- [ ] Links for each button
    - [ ] Create garden page
    - [ ] Daily task page / current garden view
    - [ ] Flower collection page
    - [ ] Flower store page

### PR DESCRIPTION
**Pull request by conalli**
## Description

add bottom navbar component - WIP

## Closes issue(s)

Closes #65 

## Screenshots

new gif with icons added

bottom-nav-2

## Changes include

- [x] New feature (non-breaking change that adds functionality)

## Checklist

- [x] I have tested this code
- [x] I have fixed visual bug

### CODE REVIEW

**Comment by kome12** [comment]
@conalli I know you're still working on this but can you also double check scrolling? Not sure if material adds this in automatically but we might have to manually add extra margin / padding at the bottom so that you can see the very bottom of the page.

File: src/components/BottomNav/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    }}));
+
+export const BottomNav = () => {
+  const [currentPage, setCurrentPage] = useState("/user/myGardens");
+  const classes = useStyles();
+  // TODO: decide how to change views on button click, fix issue with page changing
+  let history = useHistory();
+  const handlePageChange = (_event: React.ChangeEvent<{}>, newValue: string) => {
+    setCurrentPage(newValue);
+    // history.push(newValue)
```
Comment (conalli) [inline src/components/BottomNav/index.tsx:23]: Currently having issue where using history. push here requires a double tap for bottom nav to transition to new view. Not sure why...

Comment (curtisahuang) [reply src/components/BottomNav/index.tsx:23]: Is this for every transition or on first load?

**Comment by daiHash** [comment]
@conalli 
theres a conflict so please resolve it, I think its the same a had so let me know if you need help
and please resolve the conflict locally🙇‍♂️

**Review by curtisahuang — APPROVED** [review]
destroy the burger 

lgtm

File: src/components/BottomNav/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      top: 'auto',
+      bottom: 0,
+    }}));
+
+export const BottomNav = () => {
+  const [currentPage, setCurrentPage] = useState("/user/myGardens");
+  const classes = useStyles();
+  // TODO: decide how to change views on button click
+  let history = useHistory();
+  const handlePageChange = (_event: React.ChangeEvent<{}>, newValue: string) => {
```
Comment (daiHash) [inline src/components/BottomNav/index.tsx:21]: `let history = useHistory();` => `const`

File: src/components/BottomNav/index.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    setCurrentPage(newValue);
+    history.push(newValue)
+  };
+  // Fix routes when new features are added.
+  return (
```
Comment (daiHash) [inline src/components/BottomNav/index.tsx:26]: Please Add `TODO` , so easier to know that it needs to be fixed

**Review by daiHash — APPROVED** [review]
LGTM!

Please feel free to merge it after resolving the comments

**Pull request merged**